app/models/job_proxy_dispatcher.rb

```diff
@@ -13,6 +13,7 @@ def initialize
 
   def dispatch
     _dummy, t = Benchmark.realtime_block(:total_time) do
+      Benchmark.realtime_block(:container_dispatching) {dispatch_container_jobs}
```

Comment:
Average of 10 runs is 0.040116310119628906 (that is seconds probably?) there does not seem to be a difference if the queue is empty or not
15 second makes sense since a user that scheduled a scan would have to wait that amount of time for it to start.